README.md

```diff
@@ -2,21 +2,117 @@
 [![Build Status](https://travis-ci.org/thoughtbot/french_toast.svg?branch=master)](https://travis-ci.org/thoughtbot/french_toast)
 [![Code Climate](https://codeclimate.com/github/thoughtbot/french_toast/badges/gpa.svg)](https://codeclimate.com/github/thoughtbot/french_toast)
 
-Communicate information about background jobs.
+Communicate information about background jobs in Rails seamlessly.
+
+FrenchToast attempts to streamline some [progressive enhancement](https://en.wikipedia.org/wiki/Progressive_enhancement) by providing a simple and opinionated interface. In order to best take advantage of this, it is important to render that information on the backend. This allows FrenchToast to figure out the best possible communication stream, by starting at the most enhanced experience and falling back to the most basic.
```

Comment:
Does progressive enhancement start with the most basic, then build up to more advanced options if they're supported? 

`starting at the most enhanced experience and falling back to the most basic` seems to say the opposite? Maybe I'm misunderstanding...